src/index.ts

```diff
@@ -305,7 +305,8 @@ You may find the requirements here: ${cyan('https://tauri.app/v1/guides/getting-
       answers: recipeAnswers ?? {},
     });
   }
-
+  const validName =  isValidPackageName(appName) ? appName : toValidPackageName(appName) || defaults.appName
+  cfg.appName = validName
```

Comment:
`cfg.appName` is used in different places for getting the directory, can you move these two lines up before line 293

Reply:
ok, I'll update